app/controllers/api/oceans_controller.rb

```diff
@@ -1,6 +1,9 @@
 class Api::OceansController < ApplicationController
-  def index(n_oceans = 4)
-    oceans = Array.new(n_oceans) do
+  def index
+    params = Rack::Utils.parse_query URI(request.url).query
```

Comment:
this step is not needed because rails does this for you :) 
params is just there